### ISSUE TITLE
p.56 - How did you obtain the 62.5% for P(Fraud|Gift, Promo)?

### ISSUE THREAD

**Issue by markhern** [thoughtfulml/examples#5]
I only have your Ruby book and don't have your Python book yet, so maybe its covered there, but it would have been a big help to see this 62.5% explained. I guess I would need to understand what comprised the 'magical Z' as well.

**Comment by hexgnu** [comment]
Hey there,

I'll take a look at the book today and see if I can explain it. I have gotten more questions about the Naive Bayesian Chapter than any other chapter so I have revisited and revised that chapter in the new book :).

Stay tuned

**Comment by markhern** [comment]
Yes, please do give it a shot. I look forward to checking out your upcoming ML book in python as well.
